scss/_config.scss

```diff
@@ -0,0 +1,15 @@
+* {
```

Comment:
Nice idea introducing SCSS. The config file is also a good idea. In order to separate concerns, I recommend having multiple config files depending on what its configuring. For example, in this file, it seems like you're having global styles (`* { .. }`) and at the same time declaring variables. Would be great to have those separated.

Hey I have a sample SCSS project setup that you can steal. From the awesome folks at Github https://github.com/primer/css/tree/main/src